.github/workflows/cmake-multi-platform.yml

```diff
@@ -71,3 +71,33 @@ jobs:
     - name: Build
       # Build your program with the given configuration. Note that --config is needed because the default Windows generator is a multi-config generator (Visual Studio generator).
       run: cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build_type }}
+
+    - uses: benjlevesque/short-sha@v2.2
+      id: short-sha
+      with:
+        length: 6
+
+    - if: matrix.os == 'windows-latest'
+      name: Zip binaries
+      run: >
+        git archive --format=zip -o golded-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.cpp_compiler }}-${{ steps.short-sha.outputs.sha }}.zip
+        --add-file=${{ steps.strings.outputs.build-output-dir }}/golded3/golded.exe
```

Comment:
You may still use one step for windows and others if just use mask like golded* or golded.*

Reply:
Unfortunately it does not work, I tested  it locally (

Reply:
and the windows paths are different

Reply:
> You may still use one step for windows and others if just use mask like golded* or golded.*

I'm not sure golded.* will work correctly, . is just a part of a filename :)
But why golded? Isn't it named gedlnx on linux?